components/markdown-wrapper.js

```diff
@@ -1,26 +0,0 @@
-import glamorous from 'glamorous'
```

Comment:
lol, I don't even remember this thing existing 😆 

Amazing how quickly a codebase runs away from you!